### PR TITLE
AB#725 Mention manifest templates and examples

### PR DESCRIPTION
**Pull request by daniel-weisse**
Adds a short reference to the emojivoto manifest and the manifest samples/templates provided in the marblerun repo.

### CODE REVIEW

File: content/docs/workflows/define-manifest.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ weight: 1
 The Manifest is a simple JSON file that determines the key properties of your cluster: `Packages`, `Marbles`, `Secrets`, and `RecoveryKeys`.
 This article describes how to define these in your `manifest.json`.
 
+For a working example see the manifest used in our [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json), or take a look at the sample and templates provided by the [Marblerun GitHub](https://github.com/edgelesssys/marblerun/tree/master/samples).
```
Comment (thomasten) [inline content/docs/workflows/define-manifest.md:13]:
```suggestion
For a working example see the manifest of the [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json). See also the [sample and template manifests](https://github.com/edgelesssys/marblerun/tree/master/samples).
```
short sentences, less words

File: content/docs/workflows/define-manifest.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ weight: 1
 The Manifest is a simple JSON file that determines the key properties of your cluster: `Packages`, `Marbles`, `Secrets`, and `RecoveryKeys`.
 This article describes how to define these in your `manifest.json`.
 
+For a working example see the manifest of the [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json). See also the [sample and template manifests](https://github.com/edgelesssys/marblerun/tree/master/samples).
```
Comment (thomasten) [inline content/docs/workflows/define-manifest.md:13]: @flxflx @m1ghtym0 capitalize manifest or not?

Comment (daniel-weisse) [reply content/docs/workflows/define-manifest.md:13]: I think this issue needs to be discussed in detail at some point: At the moment we don't have a unified approach to whether or not we should capitalize manifest.
Both upper-, and lower-case are used throughout at docs at about the same rate.

Comment (thomasten) [reply content/docs/workflows/define-manifest.md:13]: Ok, I was under the impression that it's currently mostly uppercase.

**Review by thomasten — APPROVED** [review]

**Pull request merged**